uk.co.compucorp.civicrm.hrleaveandabsences/CRM/HRLeaveAndAbsences/Page/LeaveRequestCalendarFeed.php

```diff
@@ -19,15 +19,12 @@ public static function get() {
       $leaveFeedIcal = new LeaveRequestCalendarFeedIcal();
       $leaveFeedIcal = $leaveFeedIcal->get($leaveFeedData);
 
-      header('Content-Type: text/calendar; charset=utf-8');
-      header('Content-Disposition: attachment; filename="cal.ics"');
-      echo $leaveFeedIcal;
-      die();
+      CRM_Utils_System::download('cal.ics', 'text/calendar', $leaveFeedIcal);
+      CRM_Utils_System::civiExit();
```

Comment:
`civiExit()` is called by `download()` automatically (https://github.com/civicrm/civicrm-core/blob/fee14197b427c1781e369e5bfd36816afad6d7ee/CRM/Utils/System.php#L849-L852), so there's no need to call it here yourself